Formula/artifactory.rb

```diff
@@ -1,8 +1,8 @@
 class Artifactory < Formula
   desc "Manages binaries"
   homepage "https://www.jfrog.com/artifactory/"
-  url "https://dl.bintray.com/jfrog/artifactory/jfrog-artifactory-oss-6.23.0.zip"
-  sha256 "760302ecd706a06f9084313615a3fe7410b63ff953058cb73de2406f90dc1c6a"
+  url "https://dl.bintray.com/jfrog/artifactory/jfrog-artifactory-oss-6.23.1.zip"
+  sha256 "24c1796ede992e879e0446cf07bbb0231af6e690a03e8998ef7b7f153237fff8"
 
```

Comment:
The GNU Affero public license is at the root of the zip and each files header includes 
```
~ Artifactory is free software: you can redistribute it and/or modify
~ it under the terms of the GNU Affero General Public License as published by
~ the Free Software Foundation, either version 3 of the License, or
~ (at your option) any later version.
```

```suggestion
  license "AGPL-3.0-or-later"
```

Reply:
👍